examples/01_random_field/09_fourier_trans.py

```diff
@@ -0,0 +1,45 @@
+"""
+Generating a Transformed Periodic Random Field
+----------------------------------------------
+
+Building on the precious example, we are now going to generate periodic
+spatial random fields with a transformation applied, resulting in a level set.
+"""
+
+import numpy as np
+
+import gstools as gs
+
+# We start off by defining the spatial grid.
+L = np.array((500, 500))
+x = np.linspace(0, L[0], 300)
+y = np.linspace(0, L[1], 200)
```

Comment:
Maybe use double the period to see the periodicity in the generated field? (`2*L[0]` and `2*L[1]`)

Reply:
I've used half the domain length in y-direction.